### DISCUSSION TITLE
1.0.4

### DISCUSSION THREAD

**Discussion by rtivital** [mantinedev/mantine#52] [Releases]
Bug fixes:
- @mantine/core: Input – add option [to use tooltip inside rightSection](https://mantine.dev/core/input/#icon-and-right-section)
- @mantine/core: Modal – add option to set static modal id to avoid hydration issue
- @mantine/notifications: Fix notification onClose callback not fired when notification was closed programatically

Docs updates:
- [Notifications interactive elements examples](https://mantine.dev/others/notifications/#react-node-in-notification-message)
- [Tooltip with Input in right section](https://mantine.dev/core/input/#icon-and-right-section)
- Props table for all components are now sorted alphabetically

Environment updates:
- Storybook now works locally on Windows 10

<hr /><em>This discussion was created from the release <a href='https://github.com/mantinedev/mantine/releases/tag/1.0.4'>1.0.4</a>.</em>

**Comment by rtivital** [comment]
@cstrat @khushal87 thanks for participation in this release, I really appreciate your help and feedback.

You've been awarded with gold contributors badge! (It's just a picture though 😄)

![Gold contributors](https://aegisbadges.appspot.com/static?subject=contributor%20status&status=gold)

**Reply by khushal87** [reply]
Thanks @rtivital :D

**Reply by cstrat** [reply]
Thanks for the mention. I will stay involved as long as its helpful for you 👍🏻
